engine/jobmaster/dm/config/config.go

```diff
@@ -184,7 +184,7 @@ func FromTaskCfgs(taskCfgs []*TaskCfg) *JobCfg {
 
 // toDMTaskConfig transform a jobCfg to DM TaskCfg.
 func (c *JobCfg) toDMTaskConfig() (*dmconfig.TaskConfig, error) {
-	dmTaskCfg := &dmconfig.TaskConfig{}
+	dmTaskCfg := dmconfig.NewTaskConfig()
```

Comment:
Will it take effects at L194?

Reply:
yaml.Unmarshal should overwrite on old values